src/styles/LoginPage.module.scss

```diff
@@ -29,10 +34,15 @@
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
+    box-shadow: 2px 4px 6px $gray-light--1;
```

Comment:
again, maybe a sass variable for this one. probably won't have many styles of box-shadows, can be reused